apps-rendering/src/components/Headline/GalleryHeadline.tsx

```diff
@@ -0,0 +1,69 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	from,
+	headline,
+	neutral,
+	remSpace,
+} from '@guardian/source-foundations';
+import { grid } from 'grid/grid';
+
+// ----- Component ----- //
+
+const styles = css`
+	${headline.medium({ fontWeight: 'bold' })}
+	background-color: ${neutral[7]};
+	color: ${neutral[100]};
+	padding: 0 ${remSpace[5]} ${remSpace[9]} 0;
+	${grid.between('centre-column-start', 'centre-column-end')}
+	grid-row: 3 / 5;
+
+	${from.tablet} {
+		${grid.span('centre-column-start', 12)}
+	}
+
+	${from.desktop} {
+		${headline.xlarge({ fontWeight: 'bold' })}
+		${grid.span('centre-column-start', 8)}
+	}
+`;
+
+const backgroundStyles = css`
+	background-color: ${neutral[7]};
+	${grid.between('viewport-start', 'centre-column-end')}
+	grid-row: 3 / 5;
+
+	${from.mobileLandscape} {
+		${grid.between('viewport-start', 'viewport-end')}
+	}
+
+	${from.tablet} {
+		${grid.between('centre-column-start', 'viewport-end')}
+		margin-left: calc(${grid.columnGap} * -1/2);
+	}
+`;
+
+const standfirstBackgroundStyles = css`
+	${backgroundStyles};
+	grid-row: 5 / 7;
+	border-left: 1px solid ${neutral[20]};
+`;
+
+interface Props {
+	headline: string;
+	format: ArticleFormat;
+}
+
+const GalleryHeadline: React.FC<Props> = ({ headline }) => (
+	<>
+		<div css={backgroundStyles} />
+		<h1 css={styles}>{headline}</h1>
+		<div css={standfirstBackgroundStyles} />
```

Comment:
Would you be able to talk a little bit about what this is doing? I can see you've also introduced a `GalleryStandfirst` component separately?

Reply:
Yes, fair question. So, in the immersive headline you added another empty div that sits behind the headline text and gives it a background that stretches to the viewport end. Gallery needed the same behind the standfirst, or so I thought at the time but I can see now that it's closer to finished and I've got the correct colours everywhere I can see that this is overkill